phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexScrutinyTableOutput.java

```diff
@@ -68,14 +68,18 @@
             "    SOURCE_TS BIGINT,\n" +
             "    TARGET_TS BIGINT,\n" +
             "    HAS_TARGET_ROW BOOLEAN,\n" +
+            "    BEYOND_MAX_LOOKBACK BOOLEAN,\n" +
             "    CONSTRAINT PK PRIMARY KEY\n" +
             "    (\n" +
             "        " + SOURCE_TABLE_COL_NAME + ",\n" +
             "        " + TARGET_TABLE_COL_NAME + ",\n" +
             "        " + SCRUTINY_EXECUTE_TIME_COL_NAME + ",\n" + // time at which the scrutiny ran
             "        SOURCE_ROW_PK_HASH\n" + //  this hash makes the PK unique
             "    )\n" + // dynamic columns consisting of the source and target columns will follow
-            ")";
+            ")  COLUMN_ENCODED_BYTES = 0 "; //column encoding not supported with dyn columns (PHOENIX-5107)
```

Comment:
Do we need to do anything to disable COLUMN_ENCODIG for clusters with this table already existing?